apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/TransformerContext.java

```diff
@@ -0,0 +1,23 @@
+package gov.cms.bfd.server.war.commons;
+
+import com.codahale.metrics.MetricRegistry;
+import java.util.Optional;
+
+public class TransformerContext {
+
+  public MetricRegistry metricRegistry;
+  public Object claim;
+  public Optional<Boolean> includeTaxNumbers;
+  public FdaDrugCodeDisplayLookup drugCodeDisplayLookup;
```

Comment:
these can all be final

Reply:
I think private fields with getters is more consistent with the rest of the BFD codebase and would leave the door open for flexibility that we cannot anticipate.